### PR TITLE
feature - tabbar

### PR DESCRIPTION
**Pull request by at-datphung**
done tabbar, please review, thanks!

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/AppDelegate.swift [hunk 1]
```diff
@@ -27,6 +27,11 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         configNetwork()
         configDatabase()
+        window = UIWindow(frame: UIScreen.main.bounds)
+        window?.makeKeyAndVisible()
+
+        let vc = TabBarViewController()
+        window?.rootViewController = vc
```
Comment (at-hieutran) [inline MyApp/View/AppDelegate.swift]: line 30->34 move to func (like configNetwork)

File: MyApp/View/Controllers/TabBar/TabBarViewController.swift [hunk 2]
```diff
@@ -12,5 +12,31 @@ class TabBarViewController: UITabBarController {
...
+        let profileNavi = UINavigationController(rootViewController: profileVC)
+
+        homeNavi.tabBarItem = UITabBarItem(title: App.String.home, image: #imageLiteral(resourceName: "home"), tag: 0)
+        recentsNavi.tabBarItem = UITabBarItem(title: App.String.recents, image: #imageLiteral(resourceName: "recent"), tag: 1)
+        profileNavi.tabBarItem = UITabBarItem(title: App.String.profile, image: #imageLiteral(resourceName: "profile"), tag: 2)
+        notificationNavi.tabBarItem = UITabBarItem(title: App.String.notification, image: #imageLiteral(resourceName: "noti"), tag: 3)
+        cartNavi.tabBarItem = UITabBarItem(title: App.String.cart, image: #imageLiteral(resourceName: "cart"), tag: 4)
+
+        let controlers: [UINavigationController] = [recentsNavi, cartNavi, homeNavi, notificationNavi, profileNavi]
+        viewControllers = controlers
```
Comment (at-hieutran) [inline MyApp/View/Controllers/TabBar/TabBarViewController.swift]: homeNavi index is 2, But homeNavi.tabBarItem is 0. ????

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**